baseplate/observers/timeout.py

```diff
@@ -30,28 +29,37 @@ def from_config(cls, app_config: config.RawConfig) -> "TimeoutBaseplateObserver"
             app_config,
             {
                 "server_timeout": {
-                    "default": config.Optional(
-                        config.Timespan, default=datetime.timedelta(seconds=10)
-                    ),
+                    "default": config.Optional(config.TimespanOrInfinite, default=None),
```

Comment:
should this default to `"infinite"`?

Reply:
no, so the deprecation warning below can trigger

Reply:
ahh, makes sense